source/sync/data-model/data-model-map.txt

```diff
@@ -24,6 +24,87 @@ refer to :ref:`Development Mode <development-mode>`.
+   for the ``Canine`` collection might resemble :ref:`the example below 
+   <sync-app-services-schema-dog-example>`, where the ``title`` field of 
+   the schema is ``Dog``. That would map a Realm Database object called 
+   ``Dog`` to the ``Canine`` collection in the ``Pets`` database.
+
+   You could not have another schema whose ``title`` was ``Dog`` in the 
+   same cluster. For example, you could not sync an object with a ``title`` 
+   of ``Dog`` to both a ``Debug`` and a ``Test`` database in the 
+   same cluster. If you want to sync the same object to different 
+   collections for app development purposes, you must use different 
+   data sources - one cluster for development and a different cluster
+   for production.
+
+.. _map-realm-to-atlas-with-development-mode:
+
+Mapping with Development Mode
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+When you :ref:`enable Development Mode <enable-development-mode>` in your 
+Device Sync configuration, App Services can automatically create collections 
+and schemas for Realm Database objects that you sync. It creates those 
+collections in the database whose :ref:`name you specify 
+<sync-config-database-name>` when you enable Development Mode.
+
+With Development Mode enabled, Sync looks for a collection whose App Services 
+schema has a ``title`` that matches the name of your Realm Database object 
+type. This collection could be in any database in your linked data source. 
+It doesn't have to be in the database name you add when you configure 
+Development Mode.
+
+If there is no corresponding ``title`` in any App Services schema in your 
+linked data source, App Services creates a new collection for this object 
+type. This collection is created in the database you specify when you enable 
+Development Mode. The collection name matches the object's type, and the 
+corresponding App Services schema has a ``title`` field whose value is the 
+name of the object type. This creates the mapping between your Realm 
+Database object and the new collection.
+
+.. example::
+
+   Consider an Atlas cluster with a database named ``Pets``. It contains a 
+   ``Feline`` collection with existing data. 
+   
+   In your application code, you define a new ``Dog`` object. You enable
+   Development Mode, and specify the ``Pets`` database as the database 
+   in which to create collections for new object types. When you sync your
+   new ``Dog`` object, App Services creates a ``Dog`` collection in the 
+   ``Pets`` database, and creates a schema for it. The ``title`` of the 
+   object in the schema is ``Dog``.
+
+   If you later add a new ``Person`` object to your application code and 
+   then sync it, App Services creates a new collection for this 
+   ``Person`` object in the ``Pets`` database. If you wanted to create a 
+   collection for the ``Person`` object in a different database, you 
+   could :ref:`define a schema <enforce-a-schema>` for the ``Person`` object
```

Comment:
unrelated, but I don't love the Enforce a Schema wording in the linked page title

Reply:
I don't disagree. IIRC there may be nuance-based reasons it's "enforce" instead of "define." i.e. the act of supplying a schema forces documents in the collection to match the schema, which may not necessarily be assumed since MongoDB is a document DB and one of the things devs associate with document DBs is schemaless development or flexible schemas.